stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt

```diff
@@ -28,6 +28,11 @@ public fun Channel.isDistinctChannel(): Boolean {
     return cid.contains("!members")
 }
 
+@InternalStreamChatApi
+public fun String.isDistinctChannel(): Boolean {
```

Comment:
A bit of duplication here. I would do in the function above:
```
@InternalStreamChatApi
public fun Channel.isDistinctChannel(): Boolean {
    return cid.isDistinctChannel()
}
```
Also in `GroupChatInfoFragment`
```
private fun isDistinctChannel(): Boolean = args.cid.isDistinctChannel()
```

Reply:
Handled in a different PR.